src/lib/bootloader/sections.rb

```diff
@@ -36,7 +36,7 @@ def default=(value)
       log.info "set new default to '#{value.inspect}'"
 
       # empty value mean no default specified
-      raise "Unknown value #{value.inspect}" if !all.include?(value) && !value.empty?
+      raise "Unknown value #{value.inspect}" if !all.empty? && !all.include?(value) && !value.empty?
```

Comment:
np: In this case, I would apply the De Morgan's law:

`unless all.empty? || all.include?(value) || value.empty?`

But just a matter of preference.